1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -2,7 +2,7 @@ import { fizzBuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
 
-    test.each([3,9,42])("should return 'Fizz' when given number devisable by 3", (number) => {
+    test.each([3, 9, 42])("should return 'Fizz' when given number devisable by 3", (number) => {
```

Comment:
I think it's nice and makes your test results more readable if you add a placeholder for the example passed in using %i, like this:

```typescript
test.each([3, 9, 42])(
  "should return 'Fizz' when given number devisable by 3, such as %i",
  number => {
    expect(fizzBuzz(number)).toBe('Fizz');
  }
);`

Reply:
I agree, good spot. I had never heard of the each syntax in jest before this assignment. It's definitely been a game changer for me .